tekton-catalog/tekton-kfptask/pkg/reconciler/kfptask/reconciler.go

```diff
@@ -281,21 +324,47 @@ func dupStringMaps(source map[string]string, excludsive map[string]string) map[s
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, run *tektonv1beta1.CustomRun) reconciler.Event {
 	logger := logging.FromContext(ctx)
-	logger.Infof("Reconciling %s/%s", run.Namespace, run.Name)
+	logger.Infof("Reconciling Run %s/%s", run.Namespace, run.Name)
+	executionID := ""
+	executorInput := ""
+	PodSpecPatch := ""
```

Comment:
```suggestion
	podSpecPatch := ""
```
and change all the following references to this variable